fpdf/html.py

```diff
@@ -352,7 +359,7 @@ def __init__(
         #                    "inserted" is a special attribute indicating that a cell has be inserted in self.table_row
 
         if not tag_indents:
-            tag_indents = {}
+            tag_indents = {k: v / self.pdf.k for k, v in DEFAULT_TAG_INDENTS.items()}
         if dd_tag_indent != DEFAULT_TAG_INDENTS["dd"]:
```

Comment:
This seems to compare between different units. The user supplied argument is supposed to be in document units, but your currently store the default values in points. For the assignment below you then convert document units to points, which also doesn't looks correct.

You should probably compare against `tag_indents` instead and not convert the user input.

Reply:
> For the assignment below you then convert document units to points, which also doesn't looks correct

I'm not seeing that anywhere.

Although, it seems that I made some changes for naught tonight. More specifically, the conversion of `li_tag_indent` and `dd_tag_indent`. That was an error, as those values are only relevant if they differ from the defaults, meaning that they shouldn't be subject to this conversion. I'm going back on those changes.

As such, this note seems to not be correct:

> Those are still magic numbers (overly large now for mm), which will vary in their effect depending on the current document units

as the generated files evidently do not differ with change in document units provided that those default values in question are used.

> You should probably compare against tag_indents instead and not convert the user input

Assuming you are talking about the change in the `if` blocks for non-default `li_tag_indent` and `dd_tag_indent`, yeah, that was an error.

Reply:
> I'm not seeing that anywhere.

You actually do. Your next paragraph explains exactly what the problem is. 👍 

> As such, this note seems to not be correct:

At that time it wasn't clear to me that the value of 30 is actually in points (problem of readability!).
That works for `DEFAULT_TAG_INDENTS`, but for `li_tag_indent` and `dd_tag_indent` the default unit _is_ mm, which would result in ridiculously large indents.
Actually, since those two parameters are deprecated and their default value should never get used, it might make sense to default them to None, which will make it simpler to determine whether they have been set by the user.

Reply:
> You actually do. Your next paragraph explains exactly what the problem is

But there was conversion from points to document units, and not the other way around.  
In any case, seems like that matter is resolved now anyway.

> That works for `DEFAULT_TAG_INDENTS`, but for `li_tag_indent` and `dd_tag_indent` the default unit is mm, which would result in ridiculously large indents

Considering that we only want to deal with those if the user passes the relevant parameters, and in that case we want to handle them regardless of whether or not they are equal to relevant entries in `DEFAULT_TAG_INDENTS`, I'm making them have the value of `None` by default, so that's not going to be an issue going forward anyway.

> Actually, since those two parameters are deprecated and their default value should never get used, it might make sense to default them to None

Yeah. And there was another reason to handle them that way anyway.